pom.xml

```diff
@@ -86,7 +90,7 @@
     <properties>
         <jdk.version>1.7</jdk.version>
         <license.path>LICENSE_HEADER</license.path>
-        <mule.version>3.5.0</mule.version>
+        <mule.version>3.7.0-SNAPSHOT</mule.version>
```

Comment:
I'm not sure we should have a snapshot here. Maybe we should stick to 3.6.0 until 3.7.0 is out.